commands/user_e2e_test.go

```diff
@@ -136,3 +136,62 @@ func (s *MmctlE2ETestSuite) TestSearchUserCmd() {
 		s.Equal("Unable to find user '"+emailArg+"'", printer.GetErrorLines()[0])
 	})
 }
+
+func (s *MmctlE2ETestSuite) TestListUserCmd() {
+	s.SetupTestHelper().InitBasic()
+
+	// populate map for checking
+	userPool := []string{
+		s.th.BasicUser.Username,
+		s.th.BasicUser2.Username,
+		s.th.TeamAdminUser.Username,
+		s.th.SystemAdminUser.Username,
+	}
+	for i := 0; i < 10; i++ {
+		usr := s.th.CreateUser()
+		userPool = append(userPool, usr.Username)
+	}
+
+	s.RunForAllClients("Get some random of user", func(c client.Client) {
+		printer.Clean()
+
+		var page int
+		var all bool
+		perpage := 5
+		cmd := &cobra.Command{}
+		cmd.Flags().IntVar(&page, "page", page, "page")
+		cmd.Flags().IntVar(&perpage, "per-page", perpage, "perpage")
+		cmd.Flags().BoolVar(&all, "all", all, "all")
+
+		err := listUsersCmdF(c, cmd, []string{})
+		s.Require().Nil(err)
+		s.Require().GreaterOrEqual(len(printer.GetLines()), 5)
```

Comment:
Why the greater or equal here instead of just an `Len` check?

Reply:
it was suggested by @isacikgoz as the number of user created on initialization could increase from time to time, and may not be static

Reply:
@mgdelacroix should I change to standard `Len` check for this case?

Reply:
I am 0/5 on this. Either is fine actually.

Reply:
Apologies for the delay. Understood :)